src/inverseRobot.js

```diff
@@ -23,8 +23,19 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
   // write code here
+  const inversedRobot = {};
+
+  for (const i of Object.entries(robot)) {
```

Comment:
Object.entries is redundant here
```suggestion
  for (const key of robot) {
```